public/index.html

```diff
@@ -3,24 +3,81 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie</title>
-  </head>
-  <body>
-    <h1>Hello from Hoodie!</h1>
+    <style>
```

Comment:
Ok, maybe not totally global for now but at least it would be good to be able to reuse the styling for other UI in hoodie.